imp/NonTerminals.cs

```diff
@@ -61,298 +61,298 @@ public class NonTerminals : INonTerminals {
                                
-                               new TokenSet( /* bits: */ 0x00000000, 0x00000200, 0x00000000, /* counter: */ 1 ), /* designatorOrFuncCall */
+                               TokenSet.newFromRawData( /* bits: */ 0x00000000, 0x00000400, 0x00000000, /* counter: */ 1 ), /* designatorOrFuncCall */
                                
-                               new TokenSet( /* bits: */ 0x00000000, 0x00000000, 0x00000100, /* counter: */ 1 ), /* setValue */
+                               TokenSet.newFromRawData( /* bits: */ 0x00000000, 0x00000000, 0x00000400, /* counter: */ 1 ), /* setValue */
                                
-                               new TokenSet( /* bits: */ 0x01000000, 0x000C3E00, 0x00000110, /* counter: */ 10 ), /* element */
+                               TokenSet.newFromRawData( /* bits: */ 0x02000000, 0x00187c00, 0x00000440, /* counter: */ 10 ), /* element */
                                
-                               new TokenSet( /* bits: */ 0x00000044, 0x00000240, 0x00000000, /* counter: */ 4 ), /* formalType */
+                               TokenSet.newFromRawData( /* bits: */ 0x00000088, 0x00000480, 0x00000000, /* counter: */ 4 ), /* formalType */
                                
-                               new TokenSet( /* bits: */ 0x00000040, 0x00000040, 0x00000000, /* counter: */ 2 ), /* attributedFormalType */
+                               TokenSet.newFromRawData( /* bits: */ 0x00000080, 0x00000080, 0x00000000, /* counter: */ 2 ), /* attributedFormalType */
                                
-                               new TokenSet( /* bits: */ 0x00000040, 0x00000240, 0x00000000, /* counter: */ 3 ), /* formalParamList */
+                               TokenSet.newFromRawData( /* bits: */ 0x00000080, 0x00000480, 0x00000000, /* counter: */ 3 ), /* formalParamList */
                                
-                               new TokenSet( /* bits: */ 0x00000040, 0x00000240, 0x00000000, /* counter: */ 3 ), /* formalParams */
+                               TokenSet.newFromRawData( /* bits: */ 0x00000080, 0x00000480, 0x00000000, /* counter: */ 3 ), /* formalParams */
                                
-                               new TokenSet( /* bits: */ 0x00000040, 0x00000040, 0x00000000, /* counter: */ 2 ), /* attribFormalParams */
+                               TokenSet.newFromRawData( /* bits: */ 0x00000080, 0x00000080, 0x00000000, /* counter: */ 2 ), /* attribFormalParams */
                                
-                               new TokenSet( /* bits: */ 0x58000004, 0x00000242, 0x00000050, /* counter: */ 9 ), /* typeDeclarationTail */
+                               TokenSet.newFromRawData( /* bits: */ 0xb0000008, 0x00000484, 0x00000140, /* counter: */ 9 ), /* typeDeclarationTail */
                            };

```

Comment:
Unless I'm mistaken, this is because the ARGLIST token does not exist in the C compiler. The addition of said token early on in the enumeration shifts every following token's index in the enum by one.
Am I correct?